content/en/docs/reference/glossary/manifest.md

```diff
@@ -9,7 +9,9 @@ aka:
 tags:
 - fundamental
 ---
- Specification of a Kubernetes API object in JSON or YAML format.
+Specification of a Kubernetes API object in [JSON](https://www.json.org/json-en.html)
+or [YAML](https://yaml.org/) format.
 
 <!--more-->
-A manifest specifies the desired state of an object that Kubernetes will maintain when you apply the manifest. Each configuration file can contain multiple manifests.
+A manifest specifies the desired state of an object that Kubernetes will maintain when you apply the manifest.
+For YAML format, each file can contain multiple manifests.
```

Comment:
I don't think you can put more than one manifest in a file in JSON format, and still call that file “JSON”.

Reply:
Agreed.